stubs/zxcvbn/zxcvbn/__init__.pyi

```diff
@@ -0,0 +1,21 @@
+import datetime
+from collections.abc import Iterable
+from decimal import Decimal
+from typing_extensions import Literal, NotRequired, TypedDict
+
+from .feedback import _Feedback
+from .matching import _Match
+from .time_estimates import _CrackTimesDisplay, _CrackTimeSeconds
+
+class _Result(TypedDict):
```

Comment:
Why don't you inherit from `_TimeEstimate`, since it copies over all the data from that dict?